src/styles/blocks/card.scss

```
@@ -0,0 +1,85 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    width: 160px;
+    margin: 0 auto;
+    height: 134px;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__description {
+    margin-top: 40px;
+    font-weight: 500;
+
+    @include font-size-medium;
+  }
+  &__code {
+    color: $secondary-text-color;
+    margin-top: 4px;
+
+    @include font-size-minimal;
+  }
+
+  &__reviews {
+    @include font-size-minimal;
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price-name {
+    color: $secondary-text-color;
+
+    @include font-size-medium;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+  }
```

use scss nesting, fix it everywhere
```suggestion
  &__price {
    margin-top: 24px;
    display: flex;
    justify-content: space-between;
    
    &-name {
      // write styles here
    }
    
    &-value {
      // write styles here
    }
  }
```